### ISSUE TITLE
Looks to me like the variable does not append the necessary query parameters

### ISSUE THREAD

**Issue by rmoskal** [camunda/camunda-bpm-sdk-js#26]
This is in the variable variable resource.  The following config parameters need to be appended onto the  query string.  This is correctly done in the history resource. See below

If this is not something you can take care of immediately, I may open a pull request.

```
History.variableInstance = function(params, done) {
  if (typeof params === 'function') {
    done = arguments[0];
    params = {};
  }

  var body = {};
  var query = {};
  var queryParams = ['firstResult', 'maxResults', 'deserializeValues'];

  for (var p in params) {
    if (queryParams.indexOf(p) > -1) {
      query[p] = params[p];
    }
    else {
      body[p] = params[p];
    }
  }

  return this.http.post(this.path + '/variable-instance', {
    data: body,
    query: query,
    done: done
  });
};
```

**Comment by SebastianStamm** [comment]
Thanks for finding and reporting it! #27

**Comment by SebastianStamm** [comment]
Merged with 8d691256d0dadc1f0eac4bf0059e5c751ad709a1
